examples/shared-vpc/README.md

```diff
@@ -0,0 +1,61 @@
+# Shared VPC Example
+
+This example shows how to use this module in a Shared VPC setup. In a service project, it creates a global HTTP forwarding rule to forward traffic to an instance group. Meanwhile it ensures the host project has firewall rules in place to allow the health check ingress.
+
+## Set up the environment
+
+```
+gcloud auth application-default login
+export GOOGLE_PROJECT=$(gcloud config get-value project)
+```
+
```

Comment:
It looks like we're missing some steps here to generate a `terraform.tfvars` file to provide variables without a default value like the `host_project` and `service_project`.